Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,564 @@
+| [VTK legacy file format for vtkPolyData](https://www.vtk.org/wp-content/uploads/2015/04/file-formats.pdf) | [`itk::VTKPolyDataMeshIO`](https://itk.org/Doxygen/html/classitk_1_1VTKPolyDataMeshIO.html) | |
+
+More information on supported mesh and point set file formats is available on the
+[ITK-wasm Mesh and Point Set File Formats](https://wasm.itk.org/en/latest/introduction/file_formats/meshes.html)
+site.
+
+### What trasnform file formats can ITK import and export?
+
+ITK supports reading and writing the following transform file formats:
+
+| File Format | Read/Write | Import |
+| ----------- | ---------- | ------ |
+| InsightLegacy | [`itk::TxtTransformIOTemplate`](https://itk.org/Doxygen/html/classitk_1_1TxtTransformIOTemplate.html) | |
+| ITK HDF5 | [`itk::HDF5TransformIOTemplate`](https://itk.org/Doxygen/html/classitk_1_1HDF5TransformIOTemplate.html) | |
+| MATLAB | [`itk::MatlabTransformIOTemplate`](https://itk.org/Doxygen/html/classitk_1_1MatlabTransformIOTemplate.html) | |
+| MINC (Medical Image NetCDF) | `itk::MINCTransformIOTemplate` | |
+
+#### DICOM data
+
+##### How do I read a volume from a DICOM series?
+
+The following are alternative options for reading DICOM series. The
+first one in the list is the recommended option. The others are provided
+in case of desperation.
+
+1. Use the [itk-wasm dicom](https://itk-wasm-dicom-docs.on.fleek.co/#/),
+   JavaScript pacakge, which is available from [here](https://itk-wasm-dicom-python-docs.on.fleek.co/).
+   It has the broadest support and is the easiest to use.
+1. Use the [`itk::ImageSeriesReader`](https://itk.org/Doxygen/html/classitk_1_1ImageSeriesReader.html) in combination with the
+   `DicomSeriesFileNames`. For a full example on how to do this,
+   please look at the [`DicomSeriesReadImageWrite2.cxx`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Examples/IO/DicomSeriesReadImageWrite2.cxx)
+   code.
+1. Write a MetaImage header. This is a small text file holding
+   information about the image: spacing, dimensions, pixelt type, etc.
+   This header can hold the list of DICOM files you want to read. The
+   only restriction is that the files must be uncompressed.
+   The easy way to create a MetaImage header is to use the application:
+   `MetaImageImporter` that is available in
+   [ITKApps/MetaImageImporter](https://github.com/InsightSoftwareConsortium/ITKApps/tree/master/MetaImageImporter).
+   You can also take an existing MetaImage reader and modify it in
+   order to fit your needs. Please find attached one file that you
+   can edit it with your favorite text editor.
+1. Use `MRIConvert`: Jolinda Smith from the Lewis Center for
+   Neuroimaging at the University of Oregon developed a nice
+   application that allows you to load DICOM series and export them in
+   MetaImage and Analyze format (among others). She graciously has made
+   this application publicly available at:
+   https://darkwing.uoregon.edu/~jolinda/MRIConvert/.
```

Comment:
Perhaps we should replace with [dcm2niix](https://github.com/rordenlab/dcm2niix)?

Reply:
dcm2niix is great to convert to NIfTI, but not sure it is able to export to MetaImage, and to Analyze. Not about other tools.